### ISSUE TITLE
Tests\Feature\Api\PostEntriesTest::testPostEntries ["no filter'"] failed

### ISSUE THREAD

**Issue by jdenoc** [jdenoc/money-tracker#49]
This [test](https://github.com/jdenoc/money-tracker/blob/01d76216494215a7eae61201dace6b97bcc5243b/tests/Feature/Api/PostEntriesTest.php#L94-L119) failed with this error:

```
1) Tests\Feature\Api\PostEntriesTest::testPostEntries with data set "no filter" (array())
Expected status code 200 but received 404. Response content: []
Failed asserting that false is true.
/home/travis/build/jdenoc/money-tracker/tests/TestCase.php:66
/home/travis/build/jdenoc/money-tracker/tests/Feature/Api/PostEntriesTest.php:114
```

This is the failed [build](https://travis-ci.org/jdenoc/money-tracker/builds/308249100).

**Comment by jdenoc** [comment]
Created #50 to try and identify the cause/problem.

**Comment by jdenoc** [comment]
Fixed by #62 & #69
